Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -734,6 +913,59 @@ def custom_whitelist_api_to_user_mapper(self) -> dict[int, str]:
         """
         return reverse_dict(self.custom_whitelist_user_to_api_mapper)
 
+    @property
+    def url_action_user_to_api_mapper(self) -> dict[str, int]:
+        """Mapping the user input for URL access rule action to the API input
+        Returns:
+            dict[str, int]: Mapped dictionary.
+        """
+        return {"Pass": 1, "Alert & Deny": 2, "Continue": 3, "Deny (no log)": 4}
+
+    @property
+    def url_action_api_to_user_mapper(self) -> dict[int, str]:
+        """Mapping the API output for URL access rule action to the user output
+        Returns:
+            dict[int, str]: Mapped dictionary.
+        """
+        return reverse_dict(self.url_action_user_to_api_mapper)
+
+    @property
+    @abstractmethod
```

Comment:
Why do we need the abstactmethod decorator here? This class isnt inherited by any other class.